src/test/java/bio/terra/service/filedata/google/firestore/FireStoreDirectoryDaoTest.java

```diff
@@ -128,9 +157,7 @@ public void directoryOperationsTest() throws Exception {
 
     // Test all valid file references
     List<String> fileRefs =
-        fileObjects.stream()
-            .map(fireStoreObject -> fireStoreObject.getFileId())
-            .collect(Collectors.toList());
+        fileObjects.stream().map(FireStoreDirectoryEntry::getFileId).collect(Collectors.toList());
```

Comment:
If you prefer, you can simplify the collection thusly (ditto throughout):
```suggestion
        fileObjects.stream().map(FireStoreDirectoryEntry::getFileId).toList();
```